Gemfile.lock

```diff
@@ -102,11 +103,10 @@ GEM
       railties (>= 4.2.0)
       thor (>= 0.14, < 2.0)
     json (2.1.0)
-    json-jwt (1.8.3)
+    json-jwt (1.9.4)
       activesupport
+      aes_key_wrap
```

Comment:
Doesn't we need specific version?

Reply:
No it doesn't. See https://github.com/nov/json-jwt/blob/master/json-jwt.gemspec#L17